File: ReservationsManager/ReservationsManager.Angular/src/app/_models/Account/UserForRegisterDto.ts
import { CommonDataForRegister } from './CommonDataForRegister';

export class UserForRegisterDto extends CommonDataForRegister {
  constructor(
    name: string,
    phoneNumber: string,
    username: string,
    password: string,
    email?: string
  ) {
    super(name, phoneNumber, username, password, email);
  }
}

File: ReservationsManager/ReservationsManager.Common/Dtos/Auth/UserForRegisterDto.cs
﻿namespace ReservationsManager.Common.Dtos.Auth
{
    public class UserForRegisterDto
    {
        public string Name { get; set; }
        public string PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
    }
}
